packages/components/pie-switch/src/defs-react.ts

```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+
+export type ReactBaseType = React.InputHTMLAttributes<HTMLInputElement>
```

Comment:
I'm not entirely sure if it should use Input as base type as it is wrapped in a div and a label

Reply:
I think this is okay, we do try to have it behave as a native checkbox by exposing some properties of the inner input element

Reply:
@fernandofranca does this tell React TS users that they can apply any HTML Input attributes to the component?

Reply:
@jamieomaguire 👍 Exactly!
Important caveat, it doesn't mean such attributes will be forwarded to the implementation.